### PR TITLE
HBASE-23818 Cleanup the remaining RSGroupInfo.getTables call in the c…

### PR DESCRIPTION
**Pull request by Apache9**
…ode base

### CODE REVIEW

**Comment by Apache-HBase** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   2m  3s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +1 :green_heart: |  hbaseanti  |   0m  0s |  Patch does not have any anti-patterns.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
| +1 :green_heart: |  test4tests  |   0m  0s |  The patch appears to include 8 new or modified test files.  |
||| _ HBASE-22514 Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 36s |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |   5m 19s |  HBASE-22514 passed  |
| +1 :green_heart: |  compile  |   2m 19s |  HBASE-22514 passed  |
| +1 :green_heart: |  checkstyle  |   2m  8s |  HBASE-22514 passed  |
| +1 :green_heart: |  shadedjars  |   4m 47s |  branch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  javadoc  |   1m 30s |  HBASE-22514 passed  |
| +0 :ok: |  spotbugs  |   4m 55s |  Used deprecated FindBugs config; considering switching to SpotBugs.  |
| +1 :green_heart: |  findbugs  |   6m  2s |  HBASE-22514 passed  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 14s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  mvninstall  |   5m  6s |  the patch passed  |
| +1 :green_heart: |  compile  |   2m 20s |  the patch passed  |
| +1 :green_heart: |  javac  |   2m 20s |  the patch passed  |
| +1 :green_heart: |  checkstyle  |   2m  7s |  the patch passed  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| +1 :green_heart: |  shadedjars  |   4m 47s |  patch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  hadoopcheck  |  15m 54s |  Patch does not cause any errors with Hadoop 2.8.5 2.9.2 or 3.1.2.  |
| +1 :green_heart: |  javadoc  |   1m 30s |  the patch passed  |
| +1 :green_heart: |  findbugs  |   6m  5s |  the patch passed  |
||| _ Other Tests _ |
| +1 :green_heart: |  unit  |   0m 52s |  hbase-client in the patch passed.  |
| -1 :x: |  unit  |  52m  4s |  hbase-server in the patch failed.  |
| +1 :green_heart: |  unit  |   7m 21s |  hbase-shell in the patch passed.  |
| +1 :green_heart: |  unit  |   1m  8s |  hbase-it in the patch passed.  |
| +1 :green_heart: |  asflicense  |   1m 53s |  The patch does not generate ASF License warnings.  |
|  |   | 133m  2s |   |


| Reason | Tests |
|-------:|:------|
| Unreaped Processes | hbase-server:8 |
| Failed junit tests | hadoop.hbase.security.access.TestAccessController3 |
|   | hadoop.hbase.rsgroup.TestRSGroupsKillRS |
|   | hadoop.hbase.rsgroup.TestMigrateRSGroupInfo |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.5 Server=19.03.5 base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1152/1/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/1152 |
| Optional Tests | dupname asflicense javac javadoc unit spotbugs findbugs shadedjars hadoopcheck hbaseanti checkstyle compile |
| uname | Linux 70c70f3ad8f5 4.15.0-58-generic #64-Ubuntu SMP Tue Aug 6 11:12:41 UTC 2019 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | /home/jenkins/jenkins-slave/workspace/Base-PreCommit-GitHub-PR_PR-1152/out/precommit/personality/provided.sh |
| git revision | HBASE-22514 / d19673d82d |
| Default Java | 1.8.0_181 |
| Unreaped Processes Log | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1152/1/artifact/out/patch-unit-hbase-server-reaper.txt |
| unit | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1152/1/artifact/out/patch-unit-hbase-server.txt |
|  Test Results | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1152/1/testReport/ |
| Max. process+thread count | 9725 (vs. ulimit of 10000) |
| modules | C: hbase-client hbase-server hbase-shell hbase-it U: . |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1152/1/console |
| versions | git=2.11.0 maven=2018-06-17T18:33:14Z) findbugs=3.1.11 |
| Powered by | Apache Yetus 0.11.1 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   0m 31s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +1 :green_heart: |  hbaseanti  |   0m  0s |  Patch does not have any anti-patterns.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
| +1 :green_heart: |  test4tests  |   0m  0s |  The patch appears to include 8 new or modified test files.  |
||| _ HBASE-22514 Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 36s |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |   5m 18s |  HBASE-22514 passed  |
| +1 :green_heart: |  compile  |   2m 21s |  HBASE-22514 passed  |
| +1 :green_heart: |  checkstyle  |   2m 13s |  HBASE-22514 passed  |
| +1 :green_heart: |  shadedjars  |   4m 45s |  branch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  javadoc  |   1m 32s |  HBASE-22514 passed  |
| +0 :ok: |  spotbugs  |   4m 56s |  Used deprecated FindBugs config; considering switching to SpotBugs.  |
| +1 :green_heart: |  findbugs  |   6m  3s |  HBASE-22514 passed  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 14s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  mvninstall  |   5m  1s |  the patch passed  |
| +1 :green_heart: |  compile  |   2m 19s |  the patch passed  |
| +1 :green_heart: |  javac  |   2m 19s |  the patch passed  |
| +1 :green_heart: |  checkstyle  |   2m 13s |  the patch passed  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| +1 :green_heart: |  shadedjars  |   4m 47s |  patch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  hadoopcheck  |  16m  5s |  Patch does not cause any errors with Hadoop 2.8.5 2.9.2 or 3.1.2.  |
| +1 :green_heart: |  javadoc  |   1m 26s |  the patch passed  |
| +1 :green_heart: |  findbugs  |   6m 19s |  the patch passed  |
||| _ Other Tests _ |
| +1 :green_heart: |  unit  |   0m 55s |  hbase-client in the patch passed.  |
| +1 :green_heart: |  unit  |  67m 20s |  hbase-server in the patch passed.  |
| +1 :green_heart: |  unit  |   7m 14s |  hbase-shell in the patch passed.  |
| +1 :green_heart: |  unit  |   1m 12s |  hbase-it in the patch passed.  |
| +1 :green_heart: |  asflicense  |   2m 16s |  The patch does not generate ASF License warnings.  |
|  |   | 147m 42s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.5 Server=19.03.5 base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1152/2/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/1152 |
| Optional Tests | dupname asflicense javac javadoc unit spotbugs findbugs shadedjars hadoopcheck hbaseanti checkstyle compile |
| uname | Linux 1aae8272e8f9 4.15.0-60-generic #67-Ubuntu SMP Thu Aug 22 16:55:30 UTC 2019 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | /home/jenkins/jenkins-slave/workspace/Base-PreCommit-GitHub-PR_PR-1152/out/precommit/personality/provided.sh |
| git revision | HBASE-22514 / d19673d82d |
| Default Java | 1.8.0_181 |
|  Test Results | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1152/2/testReport/ |
| Max. process+thread count | 8824 (vs. ulimit of 10000) |
| modules | C: hbase-client hbase-server hbase-shell hbase-it U: . |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1152/2/console |
| versions | git=2.11.0 maven=2018-06-17T18:33:14Z) findbugs=3.1.11 |
| Powered by | Apache Yetus 0.11.1 https://yetus.apache.org |


This message was automatically generated.

File: hbase-server/src/main/java/org/apache/hadoop/hbase/rsgroup/RSGroupUtil.java [hunk 1]
```diff
@@ -31,9 +34,33 @@
 
   private static final Logger LOG = LoggerFactory.getLogger(RSGroupUtil.class);
 
+  public static final String RS_GROUP_ENABLED = "hbase.balancer.rsgroup.enabled";
```
Comment (saintstack) [inline hbase-server/src/main/java/org/apache/hadoop/hbase/rsgroup/RSGroupUtil.java]: Not on all the time?

Comment (Apache9) [reply hbase-server/src/main/java/org/apache/hadoop/hbase/rsgroup/RSGroupUtil.java]: You can disable the feature, then there will only be single default rs group.

**Pull request merged**